### PR TITLE
create asset css fix

### LINKED ISSUE #7341: Redesign create asset modal
**Describe the bug**
The current UI for the Create Assets modal has some CSS overflow issues. We can redesign the whole modal, like the tailwind CSS search, and make a reusable component to use across the app.

**To Reproduce**

1. Go to Assets.
2. Click on the Create Asset button.
3. Type any letter.
4. A scrollbar will appear on the whole page while a suggestion appears.

**Bug** 
https://github.com/coronasafe/care_fe/assets/70434968/a435fdf4-c3ff-432b-95ea-1b6d0f77f21e

**Expected behavior**
https://github.com/coronasafe/care_fe/assets/70434968/d34304db-ad00-462b-84a0-91661a935a85

### PR DESCRIPTION
**Pull request by MjStar24**
## Proposed Changes

- Fixes #7341
- CSS overflow fix for create asset modal

### CODE REVIEW

File: src/Components/Common/Dialog.tsx [hunk 1]
```diff
@@ -54,7 +54,7 @@ const DialogModal = (props: DialogProps) => {
                   className={classNames(
                     className,
                     fixedWidth && "w-full max-w-md",
-                    "transform rounded-2xl bg-white p-6 text-left align-middle shadow-xl transition-all"
+                    "mb-12 mt-12 transform rounded-2xl bg-white p-6 text-left align-middle shadow-xl transition-all"
```
Comment (rithviknishad) [inline src/Components/Common/Dialog.tsx:57]:
```suggestion
                    "my-12 transform rounded-2xl bg-white p-6 text-left align-middle shadow-xl transition-all"
```

**Review by rithviknishad — APPROVED** [review]

**Review by Ashesh3 — APPROVED** [review]

**Comment by nihal467** [comment]
@MjStar24 
image

- [ ] This pop-up should be in the middle of the page as current behavior

**Comment by nihal467** [comment]
@MjStar24 what is the status on this PR

**Comment by rithviknishad** [comment]
Closing as inactive + test failed + merge conflicts

**Pull request closed without merging**